packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -25,6 +26,52 @@ enum Orientation {
   landscape
 }
 
+/// Specifies a part of MediaQueryData to depend on.
+///
+/// [MediaQuery] contains a large number of related properties. Widgets frequently
+/// depend on only a few of these attributes. For example, a widget that needs to
+/// rebuild when the [MediaQueryData.textScaleFactor] changes does not need to
+/// be notified when the [MediaQueryData.size] changes.  Specifying an aspect avoids
+/// unnecessary rebuilds.
```

Comment:
I would prefer keeping this enum private (see other thread), then we don't have to worry about this doc. But if we keep it public, it should actually tell you how you would depend only on an aspect of mediaquery.